narciuch/assignment3/narciuch-3/test/WordSearchTest.java

```diff
@@ -0,0 +1,66 @@
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WordSearchTest {
+  String[] words = {"CAR", "CARD", "CART", "CAT"};
+  Dictionary dict = new Dictionary(words);
+  WordSearch wordSearch = new WordSearch();
+  char[][] grid = {{'A', 'A', 'R'}, {'T', 'C', 'D'}};
+
+  @Test
+  public void findsWordsInGrid() {
+    List<String> wordsFound = wordSearch.findWords(grid, dict);
+    assert(wordsFound.contains("CAR"));
```

Comment:
`assertTrue`.

Optional: Also please take a look at the `assert` family of methods in [`Assert` class](https://junit.org/junit4/javadoc/4.12/org/junit/Assert.html), especially `Assert.assertThat` and [`IsCollectionContaining.hasItems`](https://junit.org/junit4/javadoc/4.12/org/hamcrest/core/IsCollectionContaining.html#hasItems(T...))

If you use static imports you can get nice assertions that read almost like a sentence.